### PR TITLE
Add a way to login and sign up

### LINKED ISSUE #18: Login/Logout with Firebase
### Description

Set up a Firebase instance to support Login & Logout. 

### Resources:

List resources needed to execute the task.

### Story Points

How many points is this story worth?

### PR DESCRIPTION
**Pull request by SamuelFolledo**

### CODE REVIEW

**Review by nickKearns — APPROVED** [review]
👍

File: Petulia/Views/Authentication/LoginView.swift [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    .background(Color.accentColor)
+    .border(Color.clear, width: 2)
+    .cornerRadius(100)
+  }
+}
+
+//MARK: - Methods
+extension LoginView {
+  func loginButtonTapped() {
+//    viewModel.isLoginSuccessfully = true
```
Comment (amelinagzz) [inline Petulia/Views/Authentication/LoginView.swift]: remove?

**Review by amelinagzz — APPROVED** [review]
This PR had a lot of files to review, I'm wondering if the Alert feature alone needed its own PR. It even sounds like it could have been it's own ticket in the backlog. But other than that it looks good to go.

**Comment by amelinagzz** [comment]
This looks ready to be merged, just for the next issue you work on and this is important for good documentation of the project: make sure to include both the template of the issue and the PR template.

**Pull request merged**